setup.sh

```diff
@@ -193,7 +211,7 @@ function install () {
     echo "-------------------------"
     echo "Run the bot"
     echo "-------------------------"
-    echo "You can now use the bot by executing 'source .env/bin/activate; python3.6 freqtrade/main.py'."
+    echo "You can now use the bot by executing 'source .env/bin/activate; ${PYTHON} freqtrade/main.py'."
```

Comment:
After you have activated the virtual env, you can always just use `python` and it will use the version associated with the venv you activated.